lib/common/client/errors.rb

```diff
@@ -10,22 +10,21 @@ class ClientError < Error
         attr_accessor :status
         attr_accessor :body
 
-        def initialize(message = nil, status = nil, body = nil)
+        def initialize(error = nil, message: nil, status: nil, body: nil)
```

Comment:
Add documentation to this class

Also add/update docs in `va.gov-team` repo/README.md